travis/travis_install_nightly

```diff
@@ -108,21 +108,23 @@ fi
 # Workaround to force using system site packages (see https://github.com/Shippable/support/issues/241#issuecomment-57947925)
 rm -f $VIRTUAL_ENV/lib/python2.7/no-global-site-packages.txt
 
+echo "Installing odoo dependencies"
+# Odoo <= 7.0 don't have requirements.txt file then the 8.0 file is used by default
+if [ ! -f ${ODOO_PATH}/requirements.txt ]; then
+   wget https://raw.githubusercontent.com/odoo/odoo/8.0/requirements.txt -O ${ODOO_PATH}/requirements.txt
+fi
+# Remove python-ldap from odoo requirements because is not a common module used
+sed -i '/^python-ldap\=\=/d' ${ODOO_PATH}/requirements.txt
+# Use requests with [security] suffix to fix [Errno 111] Connection refused for old python2.7 versions.
+sed -i 's/^requests\=\=/requests[security]\=\=/g' ${ODOO_PATH}/requirements.txt
+
+pip install -q --no-binary pycparser -r ${ODOO_PATH}/requirements.txt
+
 if [ "x${WITHOUT_DEPENDENCIES}" == "x" ] ; then
     echo "Installing dependencies"
     pip install --upgrade pip
     pip install -q --no-binary pycparser -r ${HOME}/maintainer-quality-tools/requirements.txt
```

Comment:
@moylop260 
This line should be out the if sentence `if [ "x${WITHOUT_DEPENDENCIES}" == "x" ] ; then` ?